src/main/java/frc/robot/commands/Shoot.java

```diff
@@ -0,0 +1,49 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018-2019 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.GenericHID;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.OI;
+import frc.robot.RobotContainer;
+import frc.robot.subsystems.ExampleSubsystem;
+import frc.robot.subsystems.superstructure.Flywheel;
+
+public class Shoot extends CommandBase {
+	@SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
+	private final Flywheel s_flywheel;
+
+	public Shoot(Flywheel flywheel) {
+		s_flywheel = flywheel;
+		addRequirements(flywheel);
+	}
+
+	@Override
+	public void initialize() {
+	}
+
+	@Override
+	public void execute() {
+
+		if (OI.getTriggerBoolean(RobotContainer.operatorController, GenericHID.Hand.kRight)) {
+			s_flywheel.enableController();
+		} else {
+			s_flywheel.stop();
+		}
+
+	}
+
+	@Override
+	public void end(boolean interrupted) {
+	}
+
+	@Override
+	public boolean isFinished() {
+		return false;
```

Comment:
What if we did the trigger logic here? Combined with the comment I had on `end`, it should in theory work?

```suggestion
		return !OI.getTriggerBoolean(RobotContainer.operatorController, GenericHID.Hand.kRight);
```

Reply:
And then `execute` would just have the `s_flywheel.enableController();` call.